src/pages/Home/ui/Calendar/index.tsx

```diff
@@ -67,82 +80,83 @@ const Calendar = () => {
   >();
   const [currentDate, setCurrentDate] = useState({ year: 0, month: 0 });
   const calendarRef = useRef<FullCalendar | null>(null);
-  const { navigate } = useCalendarNavigation(calendarRef);
+  const [events, setEvents] = useState<CalendarEvent[]>([]);
+  const { navigate, dateRange } = useCalendarNavigation(calendarRef);
+
+  const { postScheduleListMutate } = useScheduleListMutation({
+    startDate: dateRange?.startDate || '',
+    endDate: dateRange?.endDate || '',
+    categoryIds: [3], // 추후 카테고리 id 불러오는 api 값으로 변경
+  });
+
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const convertedEvents = await postScheduleListMutate();
+        setEvents(convertedEvents);
+      } catch (error) {}
+    };
+
+    fetchEvents();
+  }, [dateRange]);
```

Comment:
```suggestion
useEffect(() => {
  postScheduleListMutate()
    .then(setEvents)
    .catch(() => {});
}, [dateRange]);
```
fetchEvents 함수를 만들지 않고도 가능합니다. 그리고 error 처리도 해주세요.